ios/cocoapods_processes.md

```diff
@@ -0,0 +1,66 @@
+# Cocoapods Processes
+
+We like to make reusable code and cocoapods is a great way to distribute that code. Individual projects may be making their own pods, but we also maintain the `Intrepid` pod within the entire iOS team. This guide aims to cover the necessary steps for creating your own pod and also maintaining one, like we must do with the `Intrepid` pod.
```

Comment:
worth maybe clarifying that 'Intrepid', is swift-wisdom and add a link. Not immediately clear.